README.md

```diff
@@ -63,7 +92,33 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-## Setup
+Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+### Building the operator
+
+```
+cd operators/go
```

Comment:
oops, sorry this is just "make build-operator"

if we cd operators/go there is no Makefile in there and it will fail. Sorry I didn't see this detail before.

Reply:
Done